env/Board.cpp

```diff
@@ -60,31 +60,32 @@ void Board::listBoards()
             while (! boardsFile.atEnd())
             {
                 QString line = QString::fromLocal8Bit(boardsFile.readLine()).trimmed();
-	        //QString line = QString::fromLocal8Bit(boardsFile.readLine().simplified());
+                
```

Comment:
There should not be any trailing spaces, you can configure your editor to do just that :)

If I merge this series, you will be encouraged to write another patch to remove them all :p